docs/DeveloperGuide.rst

```diff
@@ -0,0 +1,182 @@
+Developer Guide: StarCellBio
+============================
+
+The intent of this guide is to:
+
+    * Describe the Architecture of StarCellBio
+    * Create a Virtual Environment for developing StarCellBio
+    * Clone starcellbio_html
+    * Setup Development Environment
+    * Using watch.py to compile .soy and .gss files into .js and .css files
+    * Start the StarCellBio development server on localhost
+    * Use StarCellBio server on localhost
+    * Modify/Deploy/Use StarCellBio html_app
+    * Modify/Deploy/Use StarCellBio instructor
+    * Create/Modify/Deploy/Use StarCellBio AssignmentBuilder
+    * Create/Modify/Deploy/Use StarCellBio Lab Notebook
+
+
+Describe the Architecture of StarCellBio
+----------------------------------------
+
+    :download:`StarCellBio Architecture <StarCellBio.pdf>`
+
+Create a Virtual Environment for developing StarCellBio
+-------------------------------------------------------
+"A Virtual Environment ... is an isolated working copy of Python which allows
+you to work on a specific project without worry of affecting other project."
+
+Reference: `http://docs.python-guide.org/en/latest/dev/vertualenvs/ <http://docs.python-guide.org/en/latest/dev/vertualenvs/>`_
+
+This sample expects the ``PyVENV`` folder to contain the Virtual Environment for the sample.
```

Comment:
maybe change this to something more specific, I have a lot of venvs and something more descriptive like `starcellbio_venv` may be more helpful